### PR TITLE
Search feature

### PR DESCRIPTION
**Pull request by samlikins**
Provided a new site feature.  A Jekyll _JSON search feed_ for use with a _search page_ and _JavaScript search_ to provide **package search functionality**.  The implementation is simple, but improvements for search and result display can be enhanced.

### CODE REVIEW

File: assets/js/search.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+class packageData {
```
Comment (jwerle) [inline assets/js/search.js:1]: Could we call this `PackageData`, instead?

File: _config.yml [hunk 2]
```diff
@@ -108,7 +107,7 @@ JB :
   archive_path    : /archive/index.html
   categories_path : /packages/category
   tags_path       : /packages/tag/
-  atom_path       : /feed/atom.xml
+  ato_path        : /feed/atom.xml
```
Comment (Potherca) [inline _config.yml:110]: This looks like a typo? (As `ato`**m**`_path` is used elsewhere)

**Review by Potherca — APPROVED** [review]
The implementation looks as advertised, simple but functional.

**Comment by samlikins** [comment]
Reviewed issues and made changes. Thank you @jwerle and @Potherca.

**Pull request merged**